pkg/apis/apps/fuzzer/fuzzer.go

```diff
@@ -50,6 +50,15 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 					WhenDeleted: policies[choice&1],
 					WhenScaled:  policies[(choice>>1)&1],
 				}
+			} else {
+				// Defaulting will set empty policy fields, so they are
+				//proactively set here to avoid changes during conversion.
+				if len(s.Spec.PersistentVolumeClaimRetentionPolicy.WhenDeleted) == 0 {
```

Comment:
I looked at `types.go` and the api doc for `PersistentVolumeClaimRetentionPolicy` needs an update as well:

```go
// [...] This requires the
// StatefulSetAutoDeletePVC feature gate to be enabled, which is alpha.

PersistentVolumeClaimRetentionPolicy *StatefulSetPersistentVolumeClaimRetentionPolicy 
```

Reply:
ah, thanks for catching.